packages/protocol/contracts/test/L1/TestTaikoL1.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+//
+// ╭━━━━╮╱╱╭╮╱╱╱╱╱╭╮╱╱╱╱╱╭╮
+// ┃╭╮╭╮┃╱╱┃┃╱╱╱╱╱┃┃╱╱╱╱╱┃┃
+// ╰╯┃┃┣┻━┳┫┃╭┳━━╮┃┃╱╱╭━━┫╰━┳━━╮
+// ╱╱┃┃┃╭╮┣┫╰╯┫╭╮┃┃┃╱╭┫╭╮┃╭╮┃━━┫
+// ╱╱┃┃┃╭╮┃┃╭╮┫╰╯┃┃╰━╯┃╭╮┃╰╯┣━━┃
+// ╱╱╰╯╰╯╰┻┻╯╰┻━━╯╰━━━┻╯╰┻━━┻━━╯
+pragma solidity ^0.8.9;
+
+import "../../L1/TaikoL1.sol";
+
+contract TestTaikoL1_NoTokenomics_NoProofValidation is TaikoL1 {
+    function getConfig()
+        public
+        pure
+        override
+        returns (LibData.Config memory config)
+    {
+        config.chainId = 167;
```

Comment:
Seems we can't simply customize the L2 chain id here ( which is a common use case when deploying testnets ) , since TaikoL1 will overwrite it by `uint256 private constant L2_CHAIN_ID = 167;`? we also need to override `function getConfig()` method if we need to do so?

Reply:
Fixed